Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9-slim
```

Comment:
I think there are now newer python versions out, maybe could be upgraded

But as I said before, in general I very much doubt that python will be able to handle high volume traffic. Typically go or rust is better suited for this role.

But I guess we could have this as initial version to have at least some functionality today and we could rewrite it later